### PR TITLE
Style lists `show` page

### PR DESCRIPTION
**Pull request by andyw8**
Use the same partial as the books page for consistency

### CODE REVIEW

File: spec/features/view_an_individual_list_spec.rb [hunk 1]
```diff
@@ -19,7 +19,8 @@ def have_heading(title)
   end
 
   def have_book(title)
-    have_css("ul.books > li", text: title)
+    # TODO: rename to `.book` after #16 is merged
+    have_css(".booklist h1", text: title)
```
Comment (taylorrf) [inline spec/features/view_an_individual_list_spec.rb]: Ok, looks good to use the default books style. I think you can now rename the specs and remove the comment above.

Comment (andyw8) [inline spec/features/view_an_individual_list_spec.rb]: Done.

**Comment by taylorrf** [comment]
:+1:

**Comment by taylorrf** [comment]
I think it's close the card: https://trello.com/c/ooFxEWXm/12-displays-each-of-the-books-similar-to-how-they-are-displayed-on-the-books-page

What do you think?

**Comment by andyw8** [comment]
Ah, I didn't spot there was actually a card for this, I was just doing it make them consistent. I guess that card can marked as done then!

**Comment by taylorrf** [comment]
awesome

**Pull request merged**